### DISCUSSION TITLE
SAL Flicker Immunity (SFI) support

### DISCUSSION THREAD

**Discussion by Kangaburra** [nirnachmani/Pixie-Plus-for-Home-Assistant#10] [Q&A]
Does anyone know if the SDD400SFI dimmer would work with this integration? I might buy one and test it but thought I'd ask here first, as I note it was not one of the tested models.

**Comment by nirnachmani** [comment]
Sorry, I didn't see your question before. I have no idea if it will work or not - I put some limited support for unknown devices, assuming they are lights, but I doubt the dimmer action will be supported. Did you end up buying it and testing it?

**Comment by aszorkov** [comment]
@Kangaburra not sure if you're still holding out for this but I have SDD400's all through my place and just got this set up last night.

By default @nirnachmani's integration does exactly as they say above! They will be added as 'unknown, assumed light' and will work completely fine but only support on/off.

I got the model id from the logs so will update the config and make a pull request to get these added as known types with dimmer control later today!

**Reply by nirnachmani** [reply]
Thanks @aszorkov. You can also send me the id, model name and features and I'll add it to the code. 

It will be interesting to know if the dimming works or if they changed how they implemented the commands. The id will need to be added to the hardware_list, is_light and has_dimming in const.py
